### PR TITLE
feat: underline impacted parts of prompt

### PR DESCRIPTION
**Pull request by cabreraalex**
depends on #68 

Also show diff when editor applied.

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -9,7 +9,9 @@ module.exports = {
 	],
 	rules: {
 		// blocked by https://github.com/sveltejs/eslint-plugin-svelte/issues/652
-		'svelte/valid-compile': 'off'
+		'svelte/valid-compile': 'off',
+		'svelte/no-at-html-tags': 'off',
```
Comment (cabreraalex) [inline .eslintrc.cjs:13]: added content policy requirements to avoid issues: https://kit.svelte.dev/docs/configuration#csp

File: .eslintrc.cjs [hunk 2]
```diff
@@ -9,7 +9,9 @@ module.exports = {
...
+		'@typescript-eslint/no-unused-vars': 'off'
```
Comment (cabreraalex) [inline .eslintrc.cjs:14]: this is annoying, I have a bound variable I never read but set and this goes off.

File: .eslintrc.cjs (hunk 1 above)
Comment (cabreraalex) [reply .eslintrc.cjs:13]: JK it became a mess I didn't

Comment (Sparkier) [reply .eslintrc.cjs]: rewrote this to not use @html.

File: .eslintrc.cjs (hunk 2 above)
Comment (Sparkier) [reply .eslintrc.cjs]: rewrote this

**Comment by Sparkier** [comment]
I really like this, but I think it's still a bit buggy. For example, when a suggestion spans multiple lines.

**Comment by Sparkier** [comment]
> I really like this, but I think it's still a bit buggy. For example, when a suggestion spans multiple lines.

I think this is fixed now. Merging for now.

**Review by Sparkier — APPROVED** [review]

**Pull request merged**